.github/workflows/compile.yaml

```diff
@@ -18,17 +18,16 @@ jobs:
           submodules: 'recursive'
 
       - name: Setup Typst
-        uses: yusancky/setup-typst@v2.0.1
-        id: setup-typst
+        uses: typst-community/setup-typst@v4
         with:
-          version: 'latest'
+          typst-version: 'latest'
```

Comment:
Hi there. You may need to set `typst-version` here to specify the version to use.